config.go

```diff
@@ -537,6 +537,15 @@ type Config struct {
 	// NoDisconnectOnPongFailure controls if we'll disconnect if a peer
 	// doesn't respond to a pong in time.
 	NoDisconnectOnPongFailure bool `long:"no-disconnect-on-pong-failure" description:"If true, a peer will *not* be disconnected if a pong is not received in time or is mismatched. Defaults to false, meaning peers *will* be disconnected on pong failure."`
+
+	// UpfrontShutdownAddr specifies an address that our funds will be paid
+	// out to on cooperative channel close. This applies to all new channel
+	// opens unless overridden by an option in openchannel or by a channel
+	// acceptor.
+	// Note: If this field is set when opening a channel with a peer that
+	// does not advertise support for the upfront shutdown feature, the
+	// channel open will fail.
+	UpfrontShutdownAddr string `long:"upfront-shutdown-address" description:"Address that our funds will be paid out to on cooperative close applies to all channel opens unless overridden by openchannel options or a channel acceptor. Note: The channel open will fail if this value is set for a peer that does not support the upfront shutdown feature bit."`
```

Comment:
nit: `The address that funds will be paid out to on a cooperative channel close. This applies to all channels that are opened from the time of this option being set unless overridden for a specific channel opening. NOTE: if this option is set, any channel open will fail if the peer does not explicitly advertise the upfront-shutdown feature bit.`